### DISCUSSION TITLE
Loading SDXL models crashes Forge

### DISCUSSION THREAD

**Discussion by MeanMrKnuckles** [lllyasviel/stable-diffusion-webui-forge#391] [Q&A] [answered]
Hi, I'm on a low vram laptop with an NVIDIA GeForce RTX 3050 Ti Laptop GPU and a total VRAM 4096 MB, total RAM 7971 MB. Loading and running SD1.5 models is great, and I'm really happy with that.

However Forge crashes whenever I try to load SDXL models. 

![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/80039454/4192a85c-ea3e-4457-bcf0-7c19d7674329)

Are there settings or perquisites that I might have missed, or any logging I can turn on to see what's causing the failure?

Edited to add that I've installed using the zip file and regularly run the update batch file.

**Comment by lllyasviel** [comment, marked as answer]
https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#system-swap
same with forge

**Reply by MeanMrKnuckles** [reply]
Thanks, that's really helped. It was already set to 'Automatically manage', but I guess the max wasn't big enough. I've set it to custom size with a max of 32gb and SDXL is running with loras. Not the fastest, but I'm happy to finally be able to generate images with it.
